docs/docs/troubleshooting/recordings.md

```diff
@@ -71,3 +71,7 @@ Some users found that mounting a drive via `fstab` with the `sync` option caused
 #### Copy Times < 1 second
 
 If the storage is working quickly then this error may be caused by CPU load on the machine being too high for Frigate to have the resources to keep up. Try temporarily shutting down other services to see if the issue improves.
+
+### Infinite spinner when viewing a recording in browser
```

Comment:
The browser should emit an error if the codec isn't supported so ideally the player code would listen for this to show the error and remove the spinner. That would be more discoverable than this section of the docs so would be better IMO.

Reply:
The browser does not emit an error. That would’ve been nice.